release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2452,3 +2452,26 @@ Issued: 2021-07-27
 Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
 
 link:https://access.redhat.com/solutions/6164202[{product-title} 4.8.2 container image list]
+
+[id="ocp-4-8-3"]
+=== RHBA-2021:2896 - {product-title} 4.8.3 bug fix update
+
+Issued: 2021-08-02
+
+{product-title} release 4.8.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2157[RHBA-2021:2896] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2158[RHBA-2021:2899] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6221811[{product-title} 4.8.3 container image list]
+
+[id="ocp-4-8-3-bug-fixes"]
+==== Bug fixes
+
+* Previously, a bug in the lock implementation for the `nmstate-handler` pod caused multiple nodes to gain control. This update fixes the lock implementation so only one node is in control of the lock.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1954309[*BZ#1954309*])
+
+* Previously, there was an incorrect toleration setting on the `nmstate-handler` pod which made network configuration on nodes with the `nmstate` Operator impossible. With this update, the handler pod allows toleration on all nodes.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1960446[*BZ#1960446*])
```

Comment:
```suggestion
* Previously, a bug in the lock implementation for the `nmstate-handler` pod caused multiple nodes to gain control. This update fixes the lock implementation so that only one node is in control of the lock. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954309[*BZ#1954309*])

* Previously, there was an incorrect toleration setting on the `nmstate-handler` pod which, made network configuration on nodes with the `nmstate` Operator impossible. With this update, the handler pod allows toleration on all nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960446[*BZ#1960446*])
```

Reply:
@jeana-redhat did you intend to add the comma before the "which"?

Reply:
oops yes